dist/localforage.js

```diff
@@ -1023,14 +1017,12 @@ return /******/ (function(modules) { // webpackBootstrap
 	        LocalForage.prototype.ready = function ready(callback) {
 	            var self = this;
 
-	            var promise = new Promise(function (resolve, reject) {
-	                self._driverSet.then(function () {
-	                    if (self._ready === null) {
-	                        self._ready = self._initStorage(self._config);
-	                    }
+	            var promise = self._driverSet.then(function () {
+	                if (self._ready === null) {
+	                    self._ready = self._initDriver();
+	                }
```

Comment:
Use the _initDriver() wrapper that tries to _initStorage for the current driver and in case of an error tries the next one